bpf/lib/conntrack.h

```diff
@@ -996,6 +965,25 @@ static __always_inline int ct_create4(const void *map_main,
 	return 0;
 }
 
+static __always_inline bool
+ct_has_loopback_egress_entry4(const void *map, struct ipv4_ct_tuple *tuple,
+			      __u16 *rev_nat_index)
```

Comment:
Upstream this is guarded by `DISABLE_LOOPBACK_LB`, was there a reason why you dropped that part?

Reply:
No good reason. The bad reason was that I grepped for it in the sources and got no result, so I figured we didn't have this guard in 1.12. Trying again, I do find it, I guess I made a typo on my first attempt yesterday :facepalm:. Thanks for catching this